src/gui/qgsattributeeditorcontext.h

```diff
@@ -132,6 +133,20 @@ class GUI_EXPORT QgsAttributeEditorContext
      */
     inline QgsMapCanvas *mapCanvas() const { return mMapCanvas; }
 
+    /**
+     * Sets the associated CAD dock widget, \a cadDockWidget, (e.g. to be used in map tools).
+     * \see cadDockWidget()
+     * \since QGIS 3.10
+     */
+    inline void setCadDockWidget( QgsAdvancedDigitizingDockWidget *cadDockWidget ) { mCadDockWidget = cadDockWidget; }
```

Comment:
```suggestion
    void setCadDockWidget( QgsAdvancedDigitizingDockWidget *cadDockWidget ) { mCadDockWidget = cadDockWidget; }
```

modern compilers decide this for themselves -- the inline keyword has no effect